src/subapps/projects/containers/NewProjectContainer.tsx

```diff
@@ -14,16 +14,18 @@ const NewProjectContainer: React.FC<{}> = () => {
   const userName = useSelector(
     (state: RootState) => state.oidc.user?.profile.preferred_username
   );
-  const userOrgLabel = `${fusionConfig.personalOrgPrefix}${userName}`;
 
   const [showForm, setShowForm] = React.useState<boolean>(false);
   const [busy, setBusy] = React.useState<boolean>(false);
 
   const identities = useSelector((state: RootState) => state.auth.identities);
+
   const authenticatedIdentity = identities?.data?.identities.find(i => {
     return i['@type'] === 'Authenticated';
   });
 
+  const userOrgLabel = `${fusionConfig.personalOrgPrefix}${authenticatedIdentity?.realm}-${userName}`;
```

Comment:
Let's separate this into a lib as a function, and annotate the arguments, and use the in the other place as well